api/c/bingo-nosql/src/bingo_sim_storage.cpp

```diff
@@ -3,8 +3,7 @@
 using namespace bingo;
 using namespace indigo;
 
-SimStorage::SimStorage(int fp_size, int mt_size, int inc_size)
-    : _fingerprint_table(MMFAddress::null), _inc_size(inc_size), _mt_size(mt_size), _fp_size(fp_size)
+SimStorage::SimStorage(int fp_size, int mt_size, int inc_size) : _fingerprint_table(MMFAddress::null), _inc_size(inc_size), _mt_size(mt_size), _fp_size(fp_size)
```

Comment:
Also for initialization lists:
- Should have a freedom to put colon on the next line, 
- if absolutely impossible, then it's better to always insert new line after the colon.

So, at least, like this:
```
SimStorage::SimStorage(int fp_size, int mt_size, int inc_size) :
    _fingerprint_table(MMFAddress::null), _inc_size(inc_size), _mt_size(mt_size), _fp_size(fp_size)
```
even better like this:
```
SimStorage::SimStorage(int fp_size, int mt_size, int inc_size) : 
    _fingerprint_table(MMFAddress::null), 
    _inc_size(inc_size), 
    _mt_size(mt_size), 
    _fp_size(fp_size)
```

Reply:
Key idea of this request is to enforce same codestyle on the whole code and make automatic formatting and checking. We use clang-format and took the most similar default style with minimal changes. All other ways will lead to holy wars about code style and which of them provides better readability. So I think it's better to leave it be.